### PR TITLE
For quality of life, output the EE level in more places

### PR DESCRIPTION
**Pull request by benjamin-confino**
This will output what EE level is being tested in common build break error messages so you can immediately go to the logs for the correct server.

### CODE REVIEW

File: dev/fattest.simplicity/src/componenttest/exception/TopologyException.java [hunk 1]
```diff
@@ -12,25 +12,36 @@
...
 public class TopologyException extends Exception {
 
     public TopologyException() {
         super();
     }
 
     public TopologyException(String message, Throwable cause) {
-        super(message, cause);
+        super(getRepeatAction() + message, cause);
     }
```
Comment (Azquelt) [inline dev/fattest.simplicity/src/componenttest/exception/TopologyException.java:23]: This class will need a copyright date update.

**Review by Azquelt — COMMENTED** [review]
These changes all look sensible :+1:

The `TopologyException` is a bit odd in that we throw it on most problems in the FAT framework and in many cases it would make sense for it to have more context in the message which would need to be provided by the thrower, but as a quick fix, I think adding the repeat actions here will still be valuable.

**Review by Azquelt — APPROVED** [review]

**Comment by benjamin-confino** [comment]
#build

**Comment by benjamin-confino** [comment]
#libby

**Pull request merged**